build/ci/code_scanning/code_scan_results/filter_code_scan_results_gosec.py

```diff
@@ -0,0 +1,17 @@
+from sys import argv
+import filter_code_scan_results as filter_res
+
+
+def main():
+    output_path, target_name = tuple(argv[1:])
+
+    code_scan_res = filter_res.get_code_scan_res(output_path, target_name)
+    if not code_scan_res:
+        print("No results were found.")
+    else:
+        filter_res.filter_code_scan_res_gosec(code_scan_res)
+    filter_res.serialize_filtered_res(output_path, target_name, code_scan_res)
+
+
+if __name__ == "__main__":
+    main()
```

Comment:
Sorry for the late response, to keep it clean, please don't add any python code in this repo, please consider to use some other ways.

Reply:
This branch has been already merged. Soon I'll move code scanning out of this repo.